core/domain/topic_services.py

```diff
@@ -385,7 +385,11 @@ def _save_topic(committer_id, topic, commit_message, change_list):
         raise Exception(
             'Unexpected error: received an invalid change list when trying to '
             'save topic %s: %s' % (topic.id, change_list))
-    topic.validate()
+    topic_rights = get_topic_rights(topic.id, strict=False)
+    if topic_rights.topic_is_published:
+        topic.validate(strict=True)
+    else:
```

Comment:
Can we simplify this with `topic.validate(strict=topic_rights.topic_is_published)`?

Reply:
Done